apps/site/src/views/Resources/sections/ApiResources/ApiResources.tsx

```diff
@@ -12,7 +12,7 @@ function ApiResources() {
 		<Container>
 			<Row className={styles.row}>
 				{/* Card Component */}
-				<Col className={styles.column}>
+				<Col className={styles.column} xl={12}>
```

Comment:
Actually, noticing the API Resources card should not be in the row together with the resource cards